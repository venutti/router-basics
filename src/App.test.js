import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import App from "./App";

describe("App component", () => {
  it("ul in document", () => {
    render(<App />);

    const listElement = screen.getByRole("list");
    const listItems = screen.getAllByRole("listitem");

    expect(listElement).toBeInTheDocument();
    expect(listElement).toHaveClass("animals");
    expect(listItems.length).toBe(5);
  });
});
