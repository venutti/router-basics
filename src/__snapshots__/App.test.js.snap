// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component expect to match snapshot 1`] = `
<div>
  <div>
    <h1>
      Hello from App
    </h1>
  </div>
</div>
`;
